Vanilla TS boilerplate type error: `Module '"@anywidget/types"' has no exported member 'RenderContext'`

### Describe the bug

Scaffolding a new project using `npm create anywidget@latest` produces `widget.ts` with type error.

Opening PR with a fix soon.

### Reproduction

1. `npm create anywidget@latest`
2. Set directory name to `repro`
3. Choose Vanilla - TS template
4. `cd repro && npm i`
5. `npm run typecheck`

![Image](https://github.com/user-attachments/assets/f99e1c87-2615-4e17-9be3-04c8692ce108)

### Logs

```shell
> typecheck
> tsc --noEmit

js/widget.ts:1:15 - error TS2305: Module '"@anywidget/types"' has no exported member 'RenderContext'.

1 import type { RenderContext } from "@anywidget/types";
                ~~~~~~~~~~~~~

Found 1 error in js/widget.ts:1
```

### System Info

```shell
System:
    OS: macOS 15.3.1
    CPU: (14) arm64 Apple M3 Max
    Memory: 267.58 MB / 36.00 GB
    Shell: 5.9 - /bin/zsh
  Browsers:
    Brave Browser: 133.1.75.175
    Chrome: 134.0.6998.45
    Chrome Canary: 136.0.7064.0
    Edge: 134.0.3124.51
    Safari: 18.3

Node version: v22.14.0
NPM Version: 10.9.2

Python env info is not relevant in this case.
```

### Severity

annoyance

--- a/.changeset/ready-bars-double.md
+++ b/.changeset/ready-bars-double.md
@@ -1,0 +1,5 @@
+---
+"create-anywidget": patch
+---
+
+Import `RenderProps` instead of `RenderContext` when scaffolding vanilla TS project

--- a/packages/create-anywidget/create.js
+++ b/packages/create-anywidget/create.js
@@ -346,7 +346,7 @@ export default { render };
 /** @param {string} name */
 let widget_vanilla_ts = (name) =>
 	`\
-import type { RenderContext } from "@anywidget/types";
+import type { RenderProps } from "@anywidget/types";
 import "./widget.css";
 
 /* Specifies attributes defined with traitlets in ../src/${name}/__init__.py */
@@ -355,7 +355,7 @@ interface WidgetModel {
 	/* Add your own */
 }
 
-function render({ model, el }: RenderContext<WidgetModel>) {
+function render({ model, el }: RenderProps<WidgetModel>) {
 	let btn = document.createElement("button");
 	btn.innerHTML = \`count is \${model.get("value")}\`;
 	btn.addEventListener("click", () => {

